lib/search.js

```diff
@@ -1,6 +1,17 @@
 const bodybuilder = require('bodybuilder')
 const { MissingParamError } = require('./errors')
 
+const blacklistRoles = ['publisher', 'printer', 'consultant', 'funder',
```

Comment:
Probably worth alphabetizing this list just so it's easier to find a specific term if anyone every needs to.

Reply:
absolutely true, will do